src/RTMPStream.cpp

```diff
@@ -58,12 +58,10 @@ class ATTRIBUTE_HIDDEN CInputStreamRTMP
   void EnableStream(int streamid, bool enable) override;
   bool OpenStream(int streamid) override;
   int ReadStream(uint8_t* buffer, unsigned int bufferSize) override;
-  void PauseStream(double time) override;
+  void PauseStream(bool bPaused, double time) override;
   bool PosTime(int ms) override;
   int GetTotalTime() override { return 20; }
   int GetTime() override { return 0; }
-  bool CanPauseStream() override { return true; }
-  bool CanSeekStream() override { return true; }
```

Comment:
Think there is then also a bug on addon, there it was set to true, but here nothing https://github.com/xbmc/inputstream.rtmp/blob/627bb207f9735d60875116519762c60aa73bd412/src/RTMPStream.cpp#L125

Think it need:
```cpp
void CInputStreamRTMP::GetCapabilities(INPUTSTREAM_CAPABILITIES &caps)
{
  caps.m_mask |= INPUTSTREAM_CAPABILITIES::SUPPORTS_IPOSTIME;
  caps.m_mask |= INPUTSTREAM_CAPABILITIES::SUPPORTS_SEEK;
  caps.m_mask |= INPUTSTREAM_CAPABILITIES::SUPPORTS_PAUSE;
}
```
Not sure how this was not reported.